### PR TITLE
Add directional movement to arms

### PR DESCRIPTION
**Pull request by SjonsDePlons**
Added directional arm movement and grabbing has also been improved.

### CODE REVIEW

**Review by Jarivanoort — CHANGES_REQUESTED** [review]
Crashes

**Comment by ronanhub** [comment]
image
Was dit de bedoeling?

File: WobblyStealthGame/Assets/Scripts/Input/ActiveRagdoll/GrabBehaviour.cs [hunk 1]
```diff
@@ -6,34 +6,50 @@
...
         {
-            return;
-        } 
+            Grab(collision.rigidbody);
+        }
+    }
+
+    private void OnDisable()
+    {
+        UnGrab();
```
Comment (ahoefnagel) [inline WobblyStealthGame/Assets/Scripts/Input/ActiveRagdoll/GrabBehaviour.cs:28]:
```suggestion
        Release();
```

File: WobblyStealthGame/Assets/Scripts/Input/ActiveRagdoll/GrabBehaviour.cs [hunk 2]
```diff
@@ -6,34 +6,50 @@
...
+        
+        handJoint = gameObject.AddComponent<ConfigurableJoint>();
+        handJoint.connectedBody = objectGrab;
+        handJoint.xMotion = ConfigurableJointMotion.Locked;
+        handJoint.yMotion = ConfigurableJointMotion.Locked;
+        handJoint.zMotion = ConfigurableJointMotion.Locked;
     }
 
-    public void Grab()
+    private void UnGrab()
```
Comment (ahoefnagel) [inline WobblyStealthGame/Assets/Scripts/Input/ActiveRagdoll/GrabBehaviour.cs:45]:
```suggestion
    private void Release()
```

**Review by ahoefnagel — CHANGES_REQUESTED** [review]

**Review by ahoefnagel — APPROVED** [review]

**Pull request merged**